apis/core/lib/domain/dataset/update.ts

```diff
@@ -36,19 +36,20 @@ export async function update({
   datasetResource.deleteOut(schema.hasPart).addOut(schema.hasPart, hasPart.terms)
   datasetResource.deleteOut(cc.dimensionMetadata).addOut(cc.dimensionMetadata, dimensionMetadata.terms)
 
-  // Set LINDAS-specific contact point
+  // Set schema.org contact point
+  datasetResource.deleteOut(schema.contactPoint)
```

Comment:
There will be leftovers. Let's also remove the second level

```suggestion
  datasetResource.out(schema.contactPoint).deleteOut()
  datasetResource.deleteOut(schema.contactPoint)
```